tests/cpp-tests/Classes/NewRendererTest/NewRendererTest.cpp

```diff
@@ -888,8 +890,9 @@ cocos2d::backend::ProgramState* RendererUniformBatch::createSepiaProgramState()
     auto fileUtiles = FileUtils::getInstance();
     auto fragmentFullPath = fileUtiles->fullPathForFilename(shaderName);
     auto fragSource = fileUtiles->getStringFromFile(fragmentFullPath);
-    auto glprogram = new backend::ProgramState(positionTextureColor_vert, fragSource.c_str());
-    
+    auto program = backend::Device::getInstance()->newProgram(positionTextureColor_vert, fragSource.c_str());
+    auto glprogram = new backend::ProgramState(program);
+    CC_SAFE_RELEASE(program);
```

Comment:
glprogram -> programState

Reply:
May be need to do autorelease for program state as the method name is `createXXX`